includes/block-patterns/course/templates/video-hero.php

```diff
@@ -0,0 +1,191 @@
+<?php
+/**
+ * Video Hero pattern content.
+ *
+ * @package sensei-lms
+ */
+
```

Comment:
I think we should add the check to verify if this template is running under a WP env, right?

Like this:

```suggestion

if ( ! defined( 'ABSPATH' ) ) {
	exit; // Exit if accessed directly.
}
```

The other template files should be modified. too. =)

Reply:
Good catch!

Reply:
All updated in https://github.com/Automattic/sensei/pull/5206/commits/ebe2e14ec7d5d2c801b6d35b16467365feaae576